ansible/roles/automation/openclose_pr/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: install packages dependencies
+  dnf:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - python3-github3py
+    - PyYAML
+
+- name: install pip dependencies
+  pip:
+    name: "{{ item }}"
+    executable: pip3
+  with_items:
+    - CacheControl
+    - gitpython
+
+- name: create base directory
+  file:
+    path: "{{ basedir }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0700
+
+- name: create configuration file
+  template:
+    src: config.yml
+    dest: "{{ basedir }}/config.yml"
+
+- name: copy openclose_pr.py script
+  copy:
+    src: "{{ playbook_dir }}/../github/open_close_pr.py"
+    dest: "{{ basedir }}"
+
+- name: copy custom freeipa-pr-ci.yaml file
+  copy:
+    src: "{{ prci_config }}"
+    dest: "{{ basedir }}/{{ identifier }}.yaml"
+
+- name: copy git ssh key
+  copy:
+    src: "{{ git_sshkey }}"
+    dest: "/root/.ssh/git_ssh_key"
+    mode: 0600
+    owner: root
+    group: root
+
+- name: configure ssh to use the right ssh key when using git
+  blockinfile:
+    path: /root/.ssh/config
+    create: yes
+    block: |
+          Host github-pr_tool
+            Hostname github.com
+            IdentityFile /root/.ssh/git_ssh_key
+
+- name: clone freeipa fork
+  shell: git clone git@github-pr_tool:{{ repo_owner }}/freeipa.git {{ basedir }}/freeipa
+
+# - name: clone freeipa fork
```

Comment:
Using `git` submodule seems like to proper way to clone the repo -- why is it commented out and a `shell` module is used instead?

Reply:
I was not able to make it work with a custom hostname and ssh key. 
In my tests in last week, it got stuck. So, I just used shell since it's just git clone.